MorphicWin/App.xaml.cs

```diff
@@ -95,6 +97,19 @@ private void ConfigureServices(IServiceCollection services)
             services.AddTransient<MorphicConfigurator>();
             services.AddTransient<QuickStrip>();
             services.AddMorphicSettingsHandlers(ConfigureSettingsHandlers);
+            ConfigureCountly();
```

Comment:
If this doesn't need to do anything with `services`, I'd say move it to `OnStartup()`